readme.md

```diff
@@ -282,6 +282,8 @@ This section details services that have integrated Lighthouse data. If you're wo
 
 * **[HTTPArchive](http://httparchive.org/)** - HTTPArchive tracks how the web is built by crawling 500k pages with Web Page Test, including Lighthouse results, and stores the information in BigQuery where it is [publicly available](https://discuss.httparchive.org/t/quickstart-guide-to-exploring-the-http-archive/682).
 
+* **[Lighthouse Keeper](https://lighthouse-keeper.com/)** - Lighthouse Keeper is a free service that monitors up to 3 URLs once per day.
```

Comment:
I'm okay with this as is...buuuuuut I'd love some more context on what Lighthouse Keeper does for a user?  What does monitoring mean here?  Do I get emailed when there are regressions?  Can I set budgets?  Is it a monitoring dashboard where I can see all my sites' scores in one place?  Just a thought; otherwise, LGTM.

Reply:
I think his original did a better job of this @exterkamp 

"Lighthouse Keeper monitors your pages' Lighthouse scores and notifies you if they drop below your thresholds." SGTY as extra context?

Reply:
sgtm!